domain.md

```diff
@@ -0,0 +1,71 @@
+  **Set-Service WinRM -StartupType Automatic**  
+  **Restart-Service -Name WinRM**
+* run command to enable remote powershell: **Enable-PSRemoting -Force**
+
+
+# 3) Enable LDAPs on a on Domain Controller (DC) machine. 
+    
+Basically, there are two methods of enabling LDAPS on a DC.
+
+* Method #1: install an Enterprise Root CA  
+  The first method is the easiest: LDAPS is automatically enabled when you install an Enterprise Root CA on a Domain Controller. If you install the AD-CS role and specify the type of setup as ��Enterprise�� on a DC, all DCs in the forest will be automatically be configured to accept LDAPS.  
+    
+  
+* Method #2: add a Digital Certificate on DC  
+  Requirements for an LDAPS certificate:  
+  1) Digital Certificate must be valid for the purpose of ��Server Authentication.�� This means that they must contain the Server Authentication object identifier (OID). OIDs are like the Internet domain name space. They are series of numbers separated by dots, each with a specific meaning. For this purpose, the relevant OID we��re looking for is 1.3.6.1.5.5.7.3.1.  
+  2) The Subject name or the first name in the Subject Alternative Name (SAN) must match the Fully Qualified Domain Name (FQDN) of the host machine, such as Subject:CN=server.domain.com.  
+  3) The host machine account needs to have access to the private key. This is done when the digital certificate request is issued from that machine, or when the private key was exported and imported to a different machine.  
+    
+  The following steps setup LDAPS:  
+  1)  Create the request file.
+  2)  Submit the request to a CA. 
+  3)  Retrieve the certificate that is issued, and then save the certificate as Certnew.cer in the same folder as the request file.       Open the file in Notepad, paste the encoded certificate into the file, and then save the file.  
+      Note The saved certificate must be encoded as base64. 
+  4)  Install the issued certificate. 
+  5)  Verify that the certificate is installed in the computer's Personal store. To do this, follow these steps:  
+      Start Microsoft Management Console (MMC).  
+      Add the Certificates snap-in that manages certificates on the local computer.  
+      Expand Certificates (Local Computer), expand Personal, and then expand Certificates.  
+      A new certificate should exist in the Personal store. In the Certificate Properties dialog box, the intended purpose displayed       is Server Authentication. This certificate is issued to the computer's fully qualified host name.
+  6)  Enable the host machine account to access to the private key.  
+      The private key will be present in the following location C:\ProgramData\Microsoft\Crypto\Keys\<UniqueContainerName>  
+      Right Click the private key path and click properties --> Security and add read permissions for **NETWORK SERVICE**.
+  
+  Verifying an LDAPS connection:
+  1)  Start the Active Directory Administration Tool (Ldp.exe).
```

Comment:
Where is this available by default? How do you get it?

Reply:
in AD DC machine, once DS role installed, it is default.

Reply:
So you call ldp.exe to localhost:636 then?

Reply:
I have not try yet, but ldp.exe is there